### DISCUSSION TITLE
Deepmd potentials in pyiron

### DISCUSSION THREAD

**Discussion by kimiaGF** [pyiron/pyiron_atomistics#1020] [Q&A]
Has anyone tried to use a deepmd machine learned potential in pyiron? Are there any resources for this?

Thank you!

**Comment by jan-janssen** [comment]
I guess not yet - but as long as they use the standard `pair style` and `pair coeff` commands it should not be too difficult.

**Reply by kimiaGF** [reply]
it does use the standard commands, but the version of LAMMPS necessary for the potential to be recognized involves the installation of a plug in (see conda page for [lammps-dp](https://anaconda.org/deepmodeling/lammps-dp)). Without this plug-in enabled version of LAMMPS, running the scripts results in an "unrecognized pair potential" error.

**Reply by jan-janssen** [reply]
Now I am confused. I thought the result of the discussion https://github.com/deepmodeling/deepmd-kit-recipes/discussions/1 was that you can alternatively install `lammps` and `deepmd-kit` both from `conda-forge` to get the `pair_style deepmd`

**Comment by pmrv** [comment]
You can find documentation on [how to add custom executables for lammps](https://pyiron-atomistics.readthedocs.io/en/latest/source/installation.html#custom-executables-and-parameter-files) and [custom lammps potentials configurations](https://pyiron-atomistics.readthedocs.io/en/latest/source/faq.html#how-to-use-a-custom-potential-in-lammps).  If you are interested in adding the fitting of deepmd potentials, we have some infrastructure for fitting potentials in [pyiron_contrib](https://github.com/pyiron/pyiron_contrib/blob/c9cc894019b4dca60a179dc86d6c5c1d1c9cc577/pyiron_contrib/atomistics/ml/potentialfit.py#L21).  Over there, there are also reference implementations for MTP, ACE and EAM potentials.  Hope that helps.
